setup.py

```diff
@@ -1,24 +1,25 @@
-from setuptools import setup,find_packages
+from setuptools import setup
 
 setup(
-  name = 'Adyen',
-  packages = ['Adyen'],
-  version = '1.2.0',
-  maintainer = 'Adyen',
-  maintainer_email= 'support@adyen.com',
-  description = 'Adyen Python Api',
-  long_description = "A Python client library for accessing Adyen APIs",
-  author = 'Adyen',
-  author_email = 'support@adyen.com',
-  url = 'https://github.com/Adyen/adyen-python-api-library',
-  download_url = 'https://github.com/Adyen/adyen-python-api-library/archive/1.2.0.tar.gz',
-  keywords = ['payments', 'adyen', 'fintech'],
-  classifiers = [
-    'Development Status :: 5 - Production/Stable',
-    'Intended Audience :: Developers',
-    'Topic :: Software Development :: Libraries',
-    'License :: OSI Approved :: MIT License',
-    'Programming Language :: Python :: 2.7',
-    'Programming Language :: Python :: 3.6'
+    name='Adyen',
+    packages=['Adyen'],
+    version='1.3.0',
+    maintainer='Adyen',
+    maintainer_email='support@adyen.com',
+    description='Adyen Python Api',
+    long_description="A Python client library for accessing Adyen APIs",
+    author='Adyen',
+    author_email='support@adyen.com',
+    url='https://github.com/Adyen/adyen-python-api-library',
+    download_url='https://github.com/Adyen/'
```

Comment:
it should be 1.3.0

Reply:
what we can do is use this: https://github.com/Adyen/adyen-python-api-library/releases/latest this will always reference to the latest release. I do not think we need an actual link to a zip file right @Aleffio ?

Reply:
If provided, it must be the link to the specific zip file: https://packaging.python.org/specifications/core-metadata/?highlight=download_url#download-url

We can remove it to avoid confusion, the new pypi page will provide downloads anyway through the "Download Files" button:
https://pypi.org/project/Adyen/#description

Reply:
I will remove it as this is making it harder to release.

Reply:
done.